Source/Cotter.swift

```diff
@@ -137,13 +137,21 @@ public class Cotter {
         vc: UIViewController,
         animated: Bool,
         cb: @escaping FinalAuthCallback,
-        hideClose:Bool = false
+        hideClose: Bool = false,
+        name: String? = nil,
+        sendingMethod: String? = nil,
+        sendingDestination: String? = nil
     ) {
-        // hide the close button
+        // Hide the close button
         self.transactionPinVC.hideCloseButton = hideClose
         
         Config.instance.transactionCb = transformCb(parent: vc, cb: cb)
         
+        // Add user info if exist - name, sending method, etc.
+        if let name = name, let sendingMethod = sendingMethod, let sendingDestination = sendingDestination {
+            Config.instance.userInfo = UserInfo(name: name, sendingMethod: sendingMethod, sendingDestination: sendingDestination)
```

Comment:
I think this should be set inside the configuration parameter of Cotter initialization?

Reply:
good call